### PR TITLE
Default french locale

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 1]
```diff
@@ -412,6 +424,12 @@
     "patientSummary":{
         "translationKey":"DASHBOARD_TAB_PATIENT_SUMMARY_KEY",
         "maxRecentlyViewedPatients":10,
+        "printing": {
+            "title": "Haiti",
+            "header": "Patient Summary Report",
```
Comment (rbuisson) [inline configuration/openmrs/apps/clinical/dashboard.json:429]: 'Résumé du patient'

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 2]
```diff
@@ -3,6 +3,12 @@
         "translationKey":"DASHBOARD_TAB_GENERAL_KEY",
         "displayByDefault":true,
         "maxRecentlyViewedPatients":10,
+        "printing": {
+            "title": "Haiti",
+            "header": "Patient General Report",
```
Comment (rbuisson) [inline configuration/openmrs/apps/clinical/dashboard.json:8]: 'Rapport général'

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 1 above)
Comment (reagan-meant) [reply configuration/openmrs/apps/clinical/dashboard.json:429]: Done

File: configuration/openmrs/apps/clinical/visit.json [hunk 3]
```diff
@@ -42,9 +39,9 @@
     "dischargeSummary":{
         "translationKey": "DASHBOARD_TAB_DISCHARGE_SUMMARY_KEY",
         "printing": {
-            "title": "Bahmni",
+            "title": "Haiti",
             "header": "Discharge Summary",
```
Comment (rbuisson) [inline configuration/openmrs/apps/clinical/visit.json:43]: "Synthèse de sortie"

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 1 above)
Comment (rbuisson) [reply configuration/openmrs/apps/clinical/dashboard.json:429]: Just "Synthèse" would be good.

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 4]
```diff
@@ -217,7 +223,7 @@
                     "translationKey": "RECENT_VACCINATIONS",
                     "template": "<vaccination></vaccination>",
                     "vaccinationSets":[
-                        "Vaccination History"
+                        "Historique vaccinale"
```
Comment (rbuisson) [inline configuration/openmrs/apps/clinical/dashboard.json:226]: "Historique vaccinal"

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 5]
```diff
@@ -440,7 +458,7 @@
                         "Chief Complaint Data",
                         "Examination Notes",
                         "Smoking History",
-                        "Clief COmplaint Notes",
+                        "Clief Complaint Notes",
```
Comment (rbuisson) [inline configuration/openmrs/apps/clinical/dashboard.json:461]: Clief?

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 4 above)
Comment (reagan-meant) [reply configuration/openmrs/apps/clinical/dashboard.json:226]: Done

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 5 above)
Comment (reagan-meant) [reply configuration/openmrs/apps/clinical/dashboard.json:461]: These must have been from form one...

**Review by rbuisson — APPROVED** [review]

**Pull request merged**